pkg/controller/wavefrontupgradeutil/upgrade.go

```diff
@@ -0,0 +1,74 @@
+package wavefrontupgradeutil
```

Comment:
Can we just name this package `util`? We can add other common stuff under here over time.